### PR TITLE
chore(theme-translations): complete Polish translations

### PR DESCRIPTION
**Pull request by rev4324**

### CODE REVIEW

**Comment by Simek** [comment]
LGMT, thank you for the translation updates @rev4324! 👍 

I can also help with filling remaining gaps in i18n files, if you want to continue working on the translation.

**Comment by Josh-Cena** [comment]
Let's see if @rev4324 wants to do it. If @rev4324 doesn't have the bandwidth until next Monday (provided they did sign the CLA in the meantime) I'll merge this and you can follow up. Much appreciated, for both of you guys!

**Comment by rev4324** [comment]
No problem, I'll look into translating the remaining sentences in other files soon. Glad I could help.

**Comment by Josh-Cena** [comment]
Thanks! There are a few missing in `theme-common.json`. Mind taking a look?

**Comment by Josh-Cena** [comment]
Thanks a lot @rev4324 

@Simek We can merge once with a confirmation from you

File: packages/docusaurus-theme-translations/locales/pl/plugin-ideal-image.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "theme.IdealImageMessage.404error": "404. Image not found",
-  "theme.IdealImageMessage.error": "Error. Click to reload",
-  "theme.IdealImageMessage.load": "Click to load{sizeMessage}",
-  "theme.IdealImageMessage.loading": "Loading...",
-  "theme.IdealImageMessage.offline": "Your browser is offline. Image not loaded"
+  "theme.IdealImageMessage.404error": "404. Nie znaleziono obrazu.",
```
Comment (Simek) [inline packages/docusaurus-theme-translations/locales/pl/plugin-ideal-image.json:2]:
```suggestion
  "theme.IdealImageMessage.404error": "404. Nie znaleziono grafiki.",
```

File: packages/docusaurus-theme-translations/locales/pl/plugin-ideal-image.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+  "theme.IdealImageMessage.error": "Błąd. Kliknij by odświeżyć.",
+  "theme.IdealImageMessage.load": "Kliknij by załadować{sizeMessage}",
```
Comment (Simek) [inline packages/docusaurus-theme-translations/locales/pl/plugin-ideal-image.json:4]:
```suggestion
  "theme.IdealImageMessage.load": "Kliknij aby wczytać{sizeMessage}",
```

File: packages/docusaurus-theme-translations/locales/pl/plugin-ideal-image.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+  "theme.IdealImageMessage.loading": "Wczytywanie...",
+  "theme.IdealImageMessage.offline": "Twoja przeglądarka jest offline. Nie załadowano obrazu."
```
Comment (Simek) [inline packages/docusaurus-theme-translations/locales/pl/plugin-ideal-image.json:6]:
```suggestion
  "theme.IdealImageMessage.offline": "Twoja przeglądarka jest offline. Nie załadowano grafiki."
```

File: packages/docusaurus-theme-translations/locales/pl/theme-common.json [hunk 4]
```diff
@@ -40,18 +40,18 @@
   "theme.docs.sidebar.expandButtonAriaLabel": "Rozszerz boczny panel",
   "theme.docs.sidebar.expandButtonTitle": "Rozszerz boczny panel",
   "theme.docs.tagDocListPageTitle": "{nDocsTagged} with \"{tagName}\"",
-  "theme.docs.tagDocListPageTitle.nDocsTagged": "One doc tagged|{count} docs tagged",
-  "theme.docs.versionBadge.label": "Version: {versionLabel}",
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "Jedna strona dokumentacji otagowana|{count} stron dokumentacji otagowanych",
```
Comment (Simek) [inline packages/docusaurus-theme-translations/locales/pl/theme-common.json:43]:
```suggestion
  "theme.docs.tagDocListPageTitle.nDocsTagged": "Jedna strona dokumentacji otagowana|{count} strony dokumentacji otagowane|{count} stron dokumentacji otagowanych",
```

File: packages/docusaurus-theme-translations/locales/pl/theme-search-algolia.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
   "theme.SearchBar.label": "Szukaj",
-  "theme.SearchBar.seeAll": "See all {count} results",
+  "theme.SearchBar.seeAll": "Wyświetl wszystkie {count} wyników",
   "theme.SearchPage.algoliaLabel": "Dostawca rozwiązania Algolia",
-  "theme.SearchPage.documentsFound.plurals": "One document found|{count} documents found",
+  "theme.SearchPage.documentsFound.plurals": "Jeden dokument znaleziony|{count} dokumentów znalezione",
```
Comment (Simek) [inline packages/docusaurus-theme-translations/locales/pl/theme-search-algolia.json:5]:
```suggestion
  "theme.SearchPage.documentsFound.plurals": "Jeden dokument znaleziony|{count} dokumenty znalezione|{count} dokumentów znalezionych",
```

**Review by Simek — COMMENTED** [review]
Generally this look nice, I have suggested adding missing plural forms, Polish language uses three by default, and Docusausaurs supports it:
* https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-translations/locales/base/theme-search-algolia.json#L8

Additionally, I have suggested the alteration for the "image" translation, which suites the context a bit more, and it's more widely used on the web, ex. Google etc.

**Comment by Josh-Cena** [comment]
According to my tests, "few" is 2-4, and "many" is 5+. "other" seems to be non-integers, so it's irrelevant here. If we are already ordering the plurals in that order (one|few|many) it should be good!

**Comment by Josh-Cena** [comment]
@rev4324 Please take a look at the suggestions above ^^ I'll apply those changes and merge tomorrow if there are no further opinions.

**Comment by slorber** [comment]
If that helps: https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html

CleanShot 2022-04-21 at 12 00 06@2x

**Comment by Simek** [comment]
Just FYI, the range section is a bit overcomplicated in there, generally the range end form is used for the whole range pluralization, which also can be seen in the most right column. In other words the form of start of the range doesn't matter in i18n context.

**Comment by Josh-Cena** [comment]
> I was wondering why there are two pipes in a row in some translations (`||`).

If you are referring to the `cs` translations, that actually looks like a mistake to me because Czech only has four pluralizations, similar to Polish. I'm going to fix it.

**Comment by Simek** [comment]
Yup, it was `cs` translation for Algolia, at first I thought that they are skipping some unused forms for their language, but after our discussion here it seems like a bug. This is also only one file which uses that:
* https://cs.github.com/facebook/docusaurus?q=path%3Atranslations+language%3AJSON+%7C%7C

**Comment by Josh-Cena** [comment]
https://github.com/facebook/docusaurus/pull/7222 Thanks for spotting!

**Comment by rev4324** [comment]
Thanks for the suggestions, you're right about the "image" translation, your way fits better. Also, I didn't know about the distinction between "few" and "many", since the source that I edited had only one of these two forms.
I created a page in polish after these changes and I didn't see any incorrect/missing translations now, so I think we're good.

**Review by Josh-Cena — APPROVED** [review]
Thanks a lot! @Simek @rev4324

**Pull request merged**